simpleLooping/createListPowerOfBase.py

```diff
@@ -11,3 +11,21 @@
 # Ex:
 #   createListPowerOfBase(2, 4) # returns [1, 2, 4, 8]
 #   createListPowerOfBase(3, 3) # returns [1, 3, 9]
+from typing import List
+ 
+def createListPowerOfBase(base: int, powerCap: int) -> List[int]:
+    if powerCap == 0:
```

Comment:
You don't need to do this, if powerCap == 0 then range(powerCap) will be empty and it will make the following line generate an empty list by default.

listPowerOfBase = [base ** i for i in range(powerCap)]